### PR TITLE
correct spelling mistakes

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

**Comment by brendangregg** [comment]
[buildbot, ok to test]

**Comment by brendangregg** [comment]
[buildbot, test this please]

File: src/lua/bpf/bpf.lua [hunk 1]
```diff
@@ -475,7 +475,7 @@ end
 
 local function LOAD(dst, src, off, vtype)
 	local base = V[src].const
-	assert(base.__dissector, 'NYI: load() on variable that doesnt have dissector')
+	assert(base.__dissector, 'NYI: load() on variable that doesn't have dissector')
```
Comment (brendangregg) [inline src/lua/bpf/bpf.lua]: This breaks the build -- since the string is single quoted, adding a quote to "doesnt" breaks the quotes.

Comment (EdwardBetts) [reply src/lua/bpf/bpf.lua]: I'll fix.

**Comment by brendangregg** [comment]
[buildbot, test this please]

**Pull request merged**